internal/verifier/verifier.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 The Project Oak Authors
+// Copyright 2022-2023 The Project Oak Authors
```

Comment:
I don't think these need to be updated at all

Reply:
I have only ever seen single year. https://g3doc.corp.google.com/company/teams/opensource/copyright.md#the-year says that the multi year range is neither necessary nor encouraged, so I'd just leave everything with the initial year.